playground/backend/pkg/fs_tool/java_fs.go

```diff
@@ -0,0 +1,111 @@
+package fs_tool
+
+import (
+	"fmt"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	executableFilesFolderName = "executable_files"
+	srcFolder                 = "src"
+	binFolder                 = "bin"
+)
+
+type JavaFileSystemService struct{}
+
+func (jFS *JavaFileSystemService) PrepareFolders() error {
+	// Create executable_files/src folder
+	err := os.MkdirAll(filepath.Join(executableFilesFolderName, srcFolder), fs.ModePerm)
+	if err != nil {
+		return err
+	}
+
+	// Create executable_files/bin folder
+	err = os.MkdirAll(filepath.Join(executableFilesFolderName, binFolder), fs.ModePerm)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (jFS *JavaFileSystemService) DeletePreparedFolders() error {
+	// Delete executable_files/src folder
```

Comment:
I prefer to make documentation for the method or func instead of inline.

Reply:
I added documentation for each interface's method. Do I need to add documentation for implementations too?